rules/signing.bzl

```diff
@@ -363,7 +363,7 @@ def _hsmtool_sign(ctx, tool, digest, ecdsa_key, rsa_key, spxmsg = None, spx_key
         inputs = [digest, rsa_key.file, tool.tool] + tool.data,
         arguments = [
             "--quiet",
-            "--lockfile=/tmp/hsmtool.lock",
+            "--lockfile=/tmp/pamaury.hsmtool.lock",
```

Comment:
Seems like a local change that shouldn't be committed?

Reply:
yes